qelem/lib/screens/profile/ChangePassword.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class ChangePassword extends StatefulWidget {
+  const ChangePassword({Key? key}) : super(key: key);
+
+  @override
+  State<ChangePassword> createState() => _ChangePasswordState();
+}
+
+class _ChangePasswordState extends State<ChangePassword> {
+  bool currentHidden = true;
+  bool newHidden = true;
+  bool confirmHidden = true;
+
+  TextEditingController currentPasswordController = TextEditingController();
+  TextEditingController newPasswordController = TextEditingController();
+  TextEditingController confirmPasswordController = TextEditingController();
```

Comment:
Why do we need them?

Reply:
Yeah we don't really need them on this one.

Reply:
We need the controllers to check the similarity of the 'new password' and 'confirm password' fields.